Core/Graphics/Source/DeviceImpl.h

```diff
@@ -114,7 +117,7 @@ namespace Babylon::Graphics
 
         struct
         {
-            std::recursive_mutex Mutex{};
+            mutable std::recursive_mutex Mutex{};
```

Comment:
Is this an intentional part of this change?

Reply:
Yes, I guess I didn't need to do it, but it's weird to me that Get functions are not const, especially just to get a lock.